### ISSUE TITLE
respect `{min, max}_size` in `draw_string`

### ISSUE THREAD

**Issue by tybug** [pschanely/hypothesis-crosshair#10]
right now these parameters are ignored, which leads to crosshair generating unsound ir nodes:

```python
@given(st.characters())
def f(c):
    assert len(c) == 1
f()
```

the above gives a flaky error due to this; the root cause of invalid lengths is maybe more clear with this example:

```python
@given(st.characters())
def f(c):
    print(len(c)) # 0, 1, 2, ...
f()
```

**Comment by pschanely** [comment]
On it! Hopefully something today.

**Comment by pschanely** [comment]
Should be fixed in [0.0.6](https://pypi.org/project/hypothesis-crosshair/0.0.6/). Please reopen if it isn't!

**Comment by tybug** [comment]
This and #8 are resolved in 0.0.6, thanks! 

I think we'll be continuing this back and forth for a while longer yet; I'm grateful you're helping us iron out this interface 😄
